src/libstd/sys/unix/process/process_unix.rs

```diff
@@ -81,12 +83,13 @@ impl Command {
             match input.read(&mut bytes) {
                 Ok(0) => return Ok((p, ours)),
                 Ok(8) => {
+                    let (errno, footer) = bytes.split_at(4);
                     assert!(
-                        combine(CLOEXEC_MSG_FOOTER) == combine(&bytes[4..8]),
+                        combine(CLOEXEC_MSG_FOOTER) == combine(footer.try_into().unwrap()),
```

Comment:
Actually, why we not just compare 2 slice instead?

Reply:
Oops, that would be better indeed.